src/dashboard/cd-dashboard-news.vue

```diff
@@ -113,6 +123,30 @@
       }
     }
 
+    &__posts {
+      margin: 16px 0;
+      display: flex;
+      flex-direction: row;
+      max-width: 75%;
+
+      &-left {
+        flex-direction: column;
+        margin: 0 16px 0 16px;
+        max-width: 30%;
+      }
+
+      &-right{
+        margin: 0 16px 0 16px;
+        max-width: 70%;
+      }
+
+      &--filler {
+        background-color: @cd-very-light-grey;
+        width: 650px;
```

Comment:
This won't play nice with mobile. I _think_ the best course of action is
```css
max-width: 650px;
width: 100%;
```

That should make it not go beyond 650px, but let it shrink.